tests/client/core/test_utils.js

```diff
@@ -916,6 +917,21 @@ describe('getCompatibleVersions', () => {
     assert.equal(maxVersion, null);
     assert.equal(minVersion, null);
   });
+
+  it('should log info when OpenSearch type is found', () => {
+    const fakeLog = { info: sinon.stub() };
+    const openSearchAddon = {
+      ...fakeAddon,
+      current_version: {
+        compatibility: {},
+      },
+      type: ADDON_TYPE_OPENSEARCH,
+    };
+    getCompatibleVersions({
+      _log: fakeLog, addon: openSearchAddon, clientApp: 'firefox' });
+    assert.include(fakeLog.info.firstCall.args[0],
+      `addon is type ${ADDON_TYPE_OPENSEARCH}`);
```

Comment:
I don't think checking the log message is so important. I'd rather see this test assert the output of `getCompatibleVersions`. But you could do both.